pkg/custmsg/custom_message.go

```diff
@@ -0,0 +1,83 @@
+package custmsg
+
+import (
+	"context"
+	"fmt"
+
+	"google.golang.org/protobuf/proto"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/beholder"
+	"github.com/smartcontractkit/chainlink-common/pkg/beholder/pb"
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+)
+
+type Labeler struct {
```

Comment:
@patrickhuie19 Just fyi I shortened the naming here a bit: custmsg.Labeler hopefully stutters less while not reducing context by that much.